WordPress/src/main/java/org/wordpress/android/ui/prefs/ExperimentalFeaturesActivity.kt

```diff
@@ -76,18 +69,6 @@ class FeatureViewModel @Inject constructor(
                 AppPrefs.setManualFeatureConfig(enabled, key)
             }
         }
-
-        featureToggled(key, enabled)
-    }
-
-    private fun featureToggled(key: String, enabled: Boolean) {
-        if (key == PerAppLocaleManager.EXPERIMENTAL_PER_APP_LANGUAGE_PREF_KEY) {
-            if (enabled) {
-                perAppLocaleManager.performMigrationIfNecessary()
-            } else {
-                perAppLocaleManager.resetApplicationLocale()
```

Comment:
Minor (🔍): Consider removing this `PerAppLocaleManager.resetApplicationLocale()` function as it is no longer used.

Reply:
It's unused but very helpful during development/tesing to reset the app locale. I'll add a comment to that effect.

Reply:
Ah, cool, yea true, I remember you mentioning that and then me commenting on that, you then removing the comment, and there we go round-and-round again, apologies for the noise Nick! 😊